src/Cinematic/ArrowGenerator.cs

```diff
@@ -12,8 +12,15 @@ namespace UnityExplorer
 {
     public class ArrowGenerator
     {
-        public static GameObject CreateArrow(Vector3 arrowPosition, Quaternion arrowRotation, Color color){
+        public static GameObject CreateArrow(Vector3 arrowPosition, Quaternion arrowRotation, Color color, float arrowSizeValue = 1f)
+        {
             try {
+                Vector3 arrowSize;
+                if (arrowSizeValue > 0)
```

Comment:
```suggestion
                if (arrowSizeValue > 1)
```

So the arrow size can't go to numbers between 0 and 1 until being clamped by 1.

Reply:
if (arrowSizeValue > 0)
prevent user input negative value cause arrow direction wrong
and if user want smaller arror size can input 0.5 0.1

Reply:
Use Config no have slider seem no that problem need to manual input value